hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/properties/TracingProperties.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.web3.evm.properties;
+
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Data;
+import lombok.NonNull;
+import org.hyperledger.besu.datatypes.Address;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@ConfigurationProperties(prefix = "hedera.mirror.web3.evm.tracing")
+public class TracingProperties {
```

Comment:
Think we should not use a gerund form of the verb. Adjust other places as well.

```suggestion
@ConfigurationProperties(prefix = "hedera.mirror.web3.evm.trace")
public class TraceProperties {
```